libbpf-tools/wakeuptime.bpf.c

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0
+// Copyright (c) 2022 Nicolas Sterchele
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include "wakeuptime.h"
+#include "maps.bpf.h"
+
+#define PF_KTHREAD		0x00200000	/* kernel thread */
+#define MAX_ENTRIES		10240
```

Comment:
This is different from header file ?

Reply:
fixed in 5209cddb8a68d203a0b95eb8622150e5068c0265